ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+When filling an issue, please use the below template:
+
+###Description of the issue
```

Comment:
Add a space after the `#`s so it looks like:

```
### Description of the issue
```

Otherwise you'll get something that looks like [this](https://github.com/isunitha98selvan/coala-gedit/blob/0ea2f9da20d27abbb259f30ba17a1b11897f7542/ISSUE_TEMPLATE.md)